src/cmd/manage_nsfs.js

```diff
@@ -718,21 +717,34 @@ async function set_bucker_owner(bucket_data) {
 //// IP WHITELIST //
 ////////////////////
 
+async function _get_config_json_if_exist() {
```

Comment:
This method complete return config_data, we dont need that, Can you write a method that checks for the config file and returns true or false and use that method inside `whitelist_ips_management()`

Reply:
not relevant, resolve